modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.html

```diff
@@ -261,7 +261,7 @@ <h4 style="font-size:16px;">Validation Summary:</h4>
 </script>
 
 <script type="text/template" id="FileRowSummaryView" xmlns="http://www.w3.org/1999/html">
-    <td class='bv_fileName'><a href='<%-fileLink%>'download="<%-originalFileName%>" ><%-originalFileName%></a></td>
```

Comment:
not sure why this change was made. We do want the `download` attribute and the displayed text to be based on the originalFileName, not fileName.